### DISCUSSION TITLE
Stacking SVG top-to-bottom results in empty file

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [ImageMagick/ImageMagick#5564] [Help]
I have the AppImage downloaded from [here](https://imagemagick.org/script/download.php).
For some reason the AppImage version is 7.1.0-47, while the RPM version is 7.1.0-48.

I need to stack SVG images top-to-bottom.
I have tried the following command:
```
magick a.svg b.svg -append c.svg
```
which I found [here](https://superuser.com/a/1391092/400542), but the resulting c.svg file is empty.

**Comment by fmw42** [comment]
To write to SVG, I believe that you need a raster to vector tool such as potrace or autotrace on the system.

What SVG renderer are you using?  You should put Inkscape on your system as it is better than RSVG which is better than the Imagemagick internal MSVG renderer.

**Reply by raffaelemancuso** [reply]
Uhm how do I find out which SVG rendered I am using?

I do have both potrace and ImageMagick installed.

**Comment by fmw42** [comment]
Add -verbose to your command. See also https://imagemagick.org/Usage/draw/#svg_output
